docs/practices/environment_management.rst

```diff
@@ -0,0 +1,8 @@
+Env Management: pipx, conda
+===============================================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   pipx
```

Comment:
PyPi publishing should probably get included in this group.

Reply:
Good eye! I scanned the page on conda too quickly and did not fully register that this is about publishing to conda, not using it locally.

I'm dissolving this group and moving conda to live with PyPI Publishing in a new group called "Publishing Your Package: PyPI, Conda"